src/providers/data_provider/dp.c

```diff
@@ -184,10 +177,9 @@ dp_init_send(TALLOC_CTX *mem_ctx,
      */
     talloc_set_destructor(state->provider, dp_destructor);
 
-    subreq = sbus_server_create_and_connect_send(state->provider, ev,
-        sbus_name, NULL, sbus_address, true, 1000, uid, gid,
```

Comment:
I think `struct be_ctx` and `struct data_provider` don't need to store uid/gid anymore. This will also allow to remove it from arguments of a number of functions.

Reply:
I removed it from data provider, it is not needed because we no longer create the server. be_ctx->uid and gid are still used on other places.

Reply:
> be_ctx->uid and gid are still used on other places.

What are those places?

The thing is, 'sssd_be' (as other components) will need to be run under configured user already by 'monitor', so it will lose '--uid/--gid' command line args anyway.

Actually I had a patch removing this, imo it can be removed almost everywhere, and maybe in a couple of places `geteuid()` can be used instead.

Reply:
I see, `dp_initialized()::become_user()` uses it.
Ok, I'll get rid of `be_ctx->*id` in later PRs.